examples/common/speed_monitor_w_mfu.py

```diff
@@ -172,8 +177,8 @@ def before_dataloader(self, state: State, logger: Logger) -> None:
         self.batch_start_num_samples = int(state.timestamp.sample)
 
         # Get available GPU FLOPS
-        if not self.gpu_flops_available:
-            self.gpu_flops_available = get_gpu_flops_available(state)
+        if self.gpu_flops_available is None:
+            self.gpu_flops_available = get_gpu_flops_available(state) or 0
```

Comment:
I'm guessing that without this, the value stays `None` and keeps trying to be reset on every batch, is that right?

I think it might be a bit better to move the `or 0` to L130 so that the 0 logic is only there rather in both places?